.travis.yml

```diff
@@ -1,41 +1,24 @@
 git:
   depth: false
-jobs:
-  include:
-    - language: node_js
-      node_js:
-        - 8
-      # keep the npm cache around to speed up installs (see https://docs.npmjs.com/cli/ci)
-      cache:
-        directories:
-          - "$HOME/.npm"
-          - "$HOME/.sonar/cache"
-          - "$HOME/.sonar/native-sonar-scanner"
-      before_install:
-        - "cd optaweb-vehicle-routing-frontend"
-      # most important checks first
-      script:
-        - npm run build
-        - npm run coverage # test with coverage report
-        # check that git working tree is clean after running npm install
-        # the git command returns 1 and fails the build if there are any uncommitted changes
-        - npm install && git diff HEAD --exit-code
-      after_script: ./travis-sonarcloud.sh
-
-    - language: java
-      jdk: openjdk8
-      cache:
-        directories:
-          - "$HOME/.m2/repository"
-          - "$HOME/.m2/wrapper"
-          - "$HOME/.sonar/cache"
-      before_install:
-        - "cd optaweb-vehicle-routing-backend"
-      # Change Travis install phase to only resolve dependencies needed to run the build.
-      # Otherwise it would run `mvn install`, which results in doing some Maven phases twice (validate, compile, jar, ...).
-      # https://docs.travis-ci.com/user/languages/java/#projects-using-maven
-      #
-      # We also want to avoid Maven's install phase to prevent from writing to ~/.m2/repository, which is cached.
-      install: ./mvnw dependency:go-offline -Pjacoco,sonar --show-version
-      # do not install to avoid dirtying the cache
-      script: ./mvnw verify -Pjacoco,sonar --show-version
+language: java
+jdk: openjdk8
+cache:
+  directories:
+    - "$HOME/.m2/repository"
+    - "$HOME/.m2/wrapper"
+    - "$HOME/.npm"
+    - "$HOME/.sonar/cache"
+    - "$HOME/.sonar/native-sonar-scanner"
+before_install:
+# Change Travis install phase to only resolve dependencies needed to run the build.
+# Otherwise it would run `mvn install`, which results in doing some Maven phases twice (validate, compile, jar, ...).
+# https://docs.travis-ci.com/user/languages/java/#projects-using-maven
+#
+# We also want to avoid Maven's install phase to prevent from writing to ~/.m2/repository, which is cached.
+install: mvn dependency:go-offline -Pjacoco,sonar --show-version
+# do not install to avoid dirtying the cache
+script:
+  mvn verify -Pjacoco,sonar --show-version
+  # check that git working tree is clean after running npm install via a frontend-maven-plugin
+  # the git command returns 1 and fails the build if there are any uncommitted changes
+  git diff HEAD --exit-code
```

Comment:
I'm not YAML expert but this looks like a list of commands that are part of the `script` phase so each item should start with a dash (`-`). After latest updates no Travis build has been triggered. Maybe it's because the YAML file is not valid due to this. Not sure.